Update Proposal UI to new Mock up

Sirmorrison opened the pull request:
Fix #598 
Fix #600
Fix #683
Fix #694 
- Add governance splace screen
- Set default label colour.
- Set font weight from Bold to semibold based on mockup
- Fix html renderer indentation bug.
- Add auto sync config setting feature
- Add governance setting feature
- Add space between icon and title label on settings page

Sirmorrison committed:
Add governance splace screen
- Set default label colour.
- Set font weight from Bold to semibold based on mockup
- Remove redundant label colours
- Fix html renderer indentation bug.

Sirmorrison committed:
Implement fetch proposal functionality from splash screen
- add settings governance option.
- reset and clear proposals when governace is deactivated
- fetch proposal when governance is activated on settimgs page

Sirmorrison committed:
Add warning modal when governance is disabled
- add toast notifaction on governance toggle

Sirmorrison committed:
Add auto sync config setting feature
- Move config keys to load package

Sirmorrison committed:
add proposals overview page

Sirmorrison committed:
Add redirect to proposals list page
- use LayoutBalanceSize() for treasury balance display
- add proposal date to 'in discussion' proposal on layoutAuthorAndDate
- Add redirect to proposals details page from overview page
- Add dividing lines between proposals
- Clean up IconEditor widget
- Add search editor,
- Add proposal category and orderby dropdown
- restore sync icon with updated dimensions
- clean up code

Sirmorrison committed:
Add proposal list items
- Add back button to proposal list page

Sirmorrison committed:
Restore proposal sync listener
- clean up code

Sirmorrison committed:
fix lint error

Sirmorrison committed:
fix proposal selection bug

dreacot commented on ui/load/load.go:
```diff
@@ -213,6 +215,8 @@ func loadIcons() Icons {
 		ListGridIcon:             decredmaterial.NewImage(decredIcons["list_grid"]),
 		DecredSymbolIcon:         decredmaterial.NewImage(decredIcons["decred_symbol"]),
 		DecredSymbol2:            decredmaterial.NewImage(decredIcons["ic_decred02"]),
+		GovernanceActiveIcon:     decredmaterial.NewImage(decredIcons["governance_a"]),
```
```suggestion
		GovernanceActiveIcon:     decredmaterial.NewImage(decredIcons["governance_active"]),
```
Why isn't this governance_active

also it seems two images were added `governance_a` and `governance_active`

dreacot reviewed (changes requested):
1. This page shouldn't be on the governance page, but rather on the overview page, just the same way you have the recent transactions card with the see all button
![Screenshot from 2021-11-16 16-47-01](https://user-images.githubusercontent.com/25265396/142018136-f43f264f-4ca9-4f7a-912f-4219c734b9da.png)

2. after proposals are update there's a green checkmark that should show, it's currently not implemented
![Screenshot from 2021-11-16 16-50-13](https://user-images.githubusercontent.com/25265396/142018723-d205d9f3-c1fe-4d51-94f5-5616c001ad89.png)

3. going to the governance page, and filtering for approved proposals, the scrollbar starts at the bottom of the screen
![Screenshot from 2021-11-16 16-52-59](https://user-images.githubusercontent.com/25265396/142019143-7644810b-0402-429d-8847-344c7e564946.png)

Sirmorrison committed:
add splash screen info modal

Sirmorrison committed:
remove proposals overview page

Sirmorrison committed:
add recent proposal to overview page
- fetch only proposals that are 1 week old
- update LayoutNoProposalsFound() text
- add updated text and icon
- clean up code

Sirmorrison committed:
fix lint error

Sirmorrison committed:
Rearrange proposal filter dropdowns,
- Clean up last updated text
- Fix lint error

Sirmorrison committed:
add spacing between icon and label on more page
- Prevent proposal fetching when autosync is enable,
and proposal fetching is disabled
